gradio/utils.py

```diff
@@ -346,3 +349,222 @@ def run_coro_in_background(func: Callable, *args, **kwargs):
     """
     event_loop = asyncio.get_event_loop()
     _ = event_loop.create_task(func(*args, **kwargs))
+
+
+client = httpx.AsyncClient()
```

Comment:
Should be property of the `Request` class

Reply:
But we don't need to write `Request.client` except inside the `Request` class and perhaps for mocking tests, right? In other words, we won't need to write it out whenever we are making async requests from other parts of the `gradio` library? If that's the case, then I think it's better to put it inside the `Request` class to keep the code better organized

Reply:
Yeah, I agree, will update, thanks for defending a better-organized approach 🐱